### PR TITLE
Export ExpandDir function and add an extra return value

### PR DESCRIPTION
**Pull request by seans3**
* Exports `common.ExpandDir` function.
* Adds return value of inventory template filepath to this function, which was already being calculated.
* Adds unit tests for this newly exported function.

### CODE REVIEW

File: pkg/common/path.go [hunk 1]
```diff
@@ -137,28 +137,31 @@ func FilterInputFile(in io.Reader, tmpDir string) error {
...
+		return "", filepaths, err
 	}
+	var invFilepath string
 	for _, node := range nodes {
 		meta, err := node.GetMeta()
 		if err != nil {
 			continue
 		}
+		path := meta.Annotations[kioutil.PathAnnotation]
+		path = filepath.Join(dir, path)
```
Comment (mortent) [inline pkg/common/path.go:157]: Does this work for packages with subfolders? It might be worth adding a test for this.

File: pkg/common/path.go [hunk 2]
```diff
@@ -137,28 +137,31 @@ func FilterInputFile(in io.Reader, tmpDir string) error {
 }
 
 // expandDir takes a single package directory as a parameter, and returns
```
Comment (mortent) [inline pkg/common/path.go:139]: Update the name of the function in the comment to be capitalized.

File: pkg/common/path.go (hunk 1 above)
Comment (seans3) [reply pkg/common/path.go:157]: Added unit test for this case.

**Pull request merged**